simp_le.py

```diff
@@ -318,6 +342,10 @@ def load(self):
                 # previously
                 return self.EMPTY_DATA
             raise
+        if not content:
+            # Empty files may exist when their parent directory is not
+            # writable, or when the file mode needs to be overridden.
```

Comment:
I'm not sure what's going on here. Either inline comment needs to be improved, or something is wrong.